src/graphics/gfx_special_frames.cpp

```diff
@@ -116,3 +116,131 @@ void SpecialFrames()
     }
 }
 
+void LevelFramesNotFrozen()
+{
+    BackgroundFrameCount[26]++;
+    if(BackgroundFrameCount[26] >= 8)
+    {
+        BackgroundFrame[26]++;
+        if(BackgroundFrame[26] >= 8)
+            BackgroundFrame[26] = 0;
+        BackgroundFrameCount[26] = 0;
+    }
+    BackgroundFrameCount[18]++;
```

Comment:
Please put the space after closing the bracket to don't mess up the overlook. At here, and at other places.

Reply:
Good idea, I standardized the whole file.